packages/core/src/external/surescripts/schema/shared.ts

```
@@ -0,0 +1,238 @@
+      throw new Error(`Invalid integer: ${value}`);
+    }
+  };
+}
+
+export function toSurescriptsInteger<T extends object>(
+  key: keyof T,
+  { optional = false }: { optional?: boolean } = {}
+) {
+  return function (sourceObject: T): string {
+    const value = sourceObject[key];
+    if (typeof value === "number" && isFinite(value)) {
```

Biome (1.9.4)

[error] isFinite is unsafe. It attempts a type coercion. Use Number.isFinite instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isFinite instead.


(lint/suspicious/noGlobalIsFinite)